clippy_dev/src/main.rs

```diff
@@ -219,5 +223,15 @@ fn get_clap_config<'a>() -> ArgMatches<'a> {
                 )
                 .arg(Arg::with_name("lint").help("Which lint's page to load initially (optional)")),
         )
+        .subcommand(
+            SubCommand::with_name("lint")
+                .about("Manually run clippy on a file")
+                .arg(
+                    Arg::with_name("INPUT")
+                        .multiple(true)
```

Comment:
I don't know if I'm doing it right, but I tried multiple inputs and got an error like below:
```sh
cargo dev lint tests/ui/absurd-extreme-comparisons.rs tests/ui/bind_instead_of_map.rs    
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/clippy_dev lint tests/ui/absurd-extreme-comparisons.rs tests/ui/bind_instead_of_map.rs`
    Finished dev [unoptimized + debuginfo] target(s) in 0.07s
     Running `target/debug/clippy-driver -L ./target/debug -Z no-codegen --edition 2021 tests/ui/absurd-extreme-comparisons.rs tests/ui/bind_instead_of_map.rs`
error: multiple input filenames provided (first two filenames are `tests/ui/absurd-extreme-comparisons.rs` and `tests/ui/bind_instead_of_map.rs`)
```

Is `multiple` needed?

Reply:
It does only accept one filename, I put the `multiple` in so that you can pass through extra flags like `cargo dev lint a.rs -- --crate-type lib`. I'm not familiar with clap though so that might not be the right way to do that

I could remove it, the cases where you'd need extra flags seem pretty rare

Reply:
Hm, can you remove it? If there's a case for it, we can add it then.